src/test/init.js

```diff
@@ -26,7 +62,7 @@ test('init()', nest => {
     const mutators = ['a', 'b', 'c'].map(
       (i) => (o, { instance }) => { instance[i] = i; });
 
-    const actual = init(...mutators)();
+    const actual = init(mutators[0], mutators.slice(1, 3))();
```

Comment:
So that people can provide initializers not just as functions `init(f1, f2)` but also as an array of functions: `init([f1, f2])`, or even mix of those `init(f1, [f2, f2])`.
See README changes.